proxy/src/main/java/com/wavefront/agent/AbstractAgent.java

```diff
@@ -187,8 +187,9 @@ private void postProcessConfig() {
     // connection. the alternative is to always validate connections before reuse, but since
     // it happens fairly infrequently, and connection re-validation performance penalty is
     // incurred every time, suppressing that message seems to be a more reasonable approach.
-    org.apache.log4j.Logger.getLogger("org.apache.http.impl.execchain.RetryExec").
-        setLevel(org.apache.log4j.Level.WARN);
+    // org.apache.log4j.Logger.getLogger("org.apache.http.impl.execchain.RetryExec").
+    //     setLevel(org.apache.log4j.Level.WARN);
+    Logger.getLogger("org.apache.http.impl.execchain.RetryExec").setLevel(Level.WARNING);
```

Comment:
I don't see a level `WARNING` level in [log4j docs,](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Level.html) did you mean `WARN`?

Reply:
is java.util.logging... but i need to review if that still works, and if is need it

Reply:
In fact, I'm going to remove it, don't want to lose an error message.